app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SessionsController < ApplicationController
+  def new
+  end
```

Comment:
処理のないメソッドは下記の様に1行で記載できます。この記載だとメソッドの記入漏れで無いことが明示できます。
https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/EmptyMethod
```suggestion
  def new; end
```

Reply:
d9026e4 で修正しました。